### PR TITLE
[Nutanix Support]: Add CLI new case:test_positive_deploy_configure_by_id

### PR DESCRIPTION
**Pull request by yanpliu**
Hey,
Automation Nutanix CLI test case: verify hammer virt-who-config deploy.
Case Run: PASS
Test Results:
```
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest tests/foreman/virtwho/cli/test_nutanix.py  -k test_positive_deploy_configure_by_id -s
2022-05-13 15:57:19 - robottelo - WARNING - Dynaconf validation failed, continuing for the sake of unit tests
    server.version.release is required in env main
2022-05-13 15:57:19 - robottelo - WARNING - Dynaconf validation failed, continuing for the sake of unit tests
    server.version.release is required in env main
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.8.12, pytest-7.1.2, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/yanpliu/gitrepo/robottelo, configfile: pyproject.toml
......
.

================================================================================================ warnings summary =================================================================================================
.
.
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================================================= 1 passed, 1 deselected, 3 warnings in 193.91s (0:03:13) =============================================================================

```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

**Review by hkx303 — APPROVED** [review]
Good job @yanpliu plz remove the testing detailed log as it has the private env info

**Comment by yanpliu** [comment]
> Good job @yanpliu plz remove the testing detailed log as it has the private env info
Thanks for your reminder, done

File: robottelo/virtwho_utils.py [hunk 1]
```diff
@@ -208,7 +218,7 @@ def _get_hypervisor_mapping(logs, hypervisor_type):
     """Analysing rhsm.log and get to know: what is the hypervisor_name
     for the specific guest.
     :param str logs: the output of rhsm.log.
-    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt
+    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt. ahv
```
Comment (ogajduse) [inline robottelo/virtwho_utils.py:221]:
```suggestion
    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt, ahv
```

File: tests/foreman/virtwho/cli/test_nutanix.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+@pytest.fixture()
+def virtwho_config(form_data):
+    return VirtWhoConfig.create(form_data)['general-information']
+
+
+class TestVirtWhoConfigforNutanix:
+    @pytest.mark.tier2
+    def test_positive_deploy_configure_by_id(self, default_org, form_data, virtwho_config):
+        """Verify " hammer virt-who-config deploy"
```
Comment (Gauravtalreja1) [inline tests/foreman/virtwho/cli/test_nutanix.py:57]:
```suggestion
        """Verify "hammer virt-who-config deploy"
```

File: conf/virtwho.yaml.template [hunk 3]
```diff
@@ -58,3 +58,13 @@ VIRTWHO:
         GUEST_PORT:
         GUEST_USERNAME:
         GUEST_PASSWORD:
+    AHV:
```
[the remark this answers is not in the document]
Comment (yanpliu) [reply conf/virtwho.yaml.template:61]: > Could you pls open an MR in `satelliteqe-jenkins` repo to update this for CI?

Have already submit the PR in  `satelliteqe-jenkins` 
https://gitlab.cee.redhat.com/satelliteqe/satelliteqe-jenkins/-/merge_requests/612

File: robottelo/virtwho_utils.py [hunk 4]
```diff
@@ -35,10 +35,10 @@ def get_system(system_type):
     """Return a dict account for ssh connect.
 
     :param str system_type: The type of the system, should be one of
-        ('satellite', 'esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt').
+        ('satellite', 'esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt', 'ahv').
     :raises: VirtWhoError: If wrong ``system_type`` specified.
     """
-    if system_type in ['esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt']:
+    if system_type in ['esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt', 'ahv']:
```
[the remark this answers is not in the document]
Comment (yanpliu) [reply robottelo/virtwho_utils.py:41]: > Optional, here you can define a variable for this list of `system_types`, which can be reused later in the test

Thanks for your suggestion ,will update it

File: robottelo/virtwho_utils.py (hunk 1 above)
Comment (yanpliu) [reply robottelo/virtwho_utils.py]: > 
Have already submit the update. Please confirm it thanks.

File: robottelo/virtwho_utils.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (yanpliu) [reply robottelo/virtwho_utils.py]: > > Optional, here you can define a variable for this list of `system_types`, which can be reused later in the test
> 
> Thanks for your suggestion ,will update it

Have already submit the changes,  please review it ,thanks.

File: robottelo/virtwho_utils.py [hunk 5]
```diff
@@ -54,7 +54,17 @@ def get_system(system_type):
     else:
         raise VirtWhoError(
             '"{}" system type is not supported. Please use one of {}'.format(
-                system_type, ('satellite', 'esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt')
```
[the remark this answers is not in the document]
Comment (yanpliu) [reply robottelo/virtwho_utils.py]: > Could you use f-strings here?

Have already submit the change, please review it ,thanks

**Comment by yanpliu** [comment]
@Gauravtalreja1 All the changes have been updated, could you please have a review and merge the PR? Thank you very much!

**Review by Gauravtalreja1 — APPROVED** [review]
ACK

**Comment by yanpliu** [comment]
@Gauravtalreja1 There are three people have reviewed the PR, could you please merge this PR? Thank you very much!

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**